plugins/available/pack.plugin.bash

```diff
@@ -49,7 +50,7 @@ __pack_handle_reply() {
 			else
 				allflags=("${flags[*]} ${two_word_flags[*]}")
 			fi
-			COMPREPLY=($(compgen -W "${allflags[*]}" -- "$cur"))
+			IFS=" " read -r -a COMPREPLY <<< "$(compgen -W "${allflags[*]}" -- "$cur")"
```

Comment:
This doesn't work. `compgen` produces the results with multiple lines (where each line contains one completion candidate), while `read` will read **only the first line** and split the first line with `IFS`.

Reply:
https://github.com/Bash-it/bash-it/blob/14ee27c8a8aaa8ca5255676fde7525074825e58b/completion/available/sqlmap.completion.bash#L21-L73 has the same issue.

One may specify `-d ''` to `read` to change the line separator to NUL since the existing versions of Bash anyway filters NUL out from the result of the command substitutions `$(...)`.